src/main/java/de/tum/in/www1/artemis/web/rest/OrganizationResource.java

```diff
@@ -51,14 +53,15 @@ public OrganizationResource(OrganizationService organizationService, Organizatio
     }
 
     /**
-     * POST /organizations/course/:courseId/organization/:organizationId :
+     * POST organizations/course/:courseId/organization/:organizationId :
      * Add a course to an organization
      *
      * @param courseId the id of the course to add
      * @param organizationId the id of the organization where the course should be added
      * @return empty ResponseEntity with status 200 (OK), or 404 (Not Found) otherwise
      */
-    @PostMapping("/organizations/course/{courseId}/organization/{organizationId}")
+    @PostMapping("organizations/course/{courseId}/organization/{organizationId}") // TODO: this path doesn't make sense, should be organizations/{organizationId}/courses, the
```

Comment:
You have changed all other REST Calls, so I suggest you improve this one as well